tests/telemetry/test_telemetry.py

```diff
@@ -60,3 +70,20 @@ def test_telemetry_enabledbydefault(duthost):
         if str(k) == "status":
             status_expected = "enabled";
             pytest_assert(str(v) == status_expected, "Telemetry feature is not enabled")
+
+def test_telemetry_ouput(duthost, ptfhost):
+    """Run pyclient from ptfdocker and show gnmi server outputself.
+    """
+    logger.info('start telemetry output testing')
+    setup_telemetry_forpyclient(duthost)
+    # wait till telemetry is restarted
+    pytest_assert(wait_until(100, 10, duthost.is_service_fully_started, "telemetry"), "TELEMETRY not started")
+    logger.info('telemetry process restarted. Now run pyclient on ptfdocker')
+    dut_ip = duthost.setup()['ansible_facts']['ansible_eth0']['ipv4']['address']
+    # pyclient should be available on ptfhost. If not fail pytest.
+    file_exists = ptfhost.stat(path="/gnxi/gnmi_cli_py/py_gnmicli.py")
+    pytest_assert(file_exists["stat"]["exists"] is True)
+    cmd = 'python /gnxi/gnmi_cli_py/py_gnmicli.py -g -t {0} -p 50051 -m get -x COUNTERS/Ethernet0 -xt COUNTERS_DB -o "ndastreamingservertest"'.format(dut_ip)
+    show_gnmi_out = ptfhost.shell(cmd)['stdout']
```

Comment:
how do we know if the command has returned success?

Reply:
added some further checks.